corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -80,6 +82,20 @@ def hash_doc_ids(doc_ids):
             rows = fetchall_as_namedtuple(cursor)
             return {row.doc_id: row.hash for row in rows}
 
+    @staticmethod
+    def hash_doc_ids_python(doc_ids):
+        key = b'\x00' * 16
```

Comment:
maybe a comment on this function, i'm not really sure why this is necessary. is it trying to create 16 bytes?